src/styles/blocks/header.css

```
@@ -16,7 +16,9 @@
     }
 
     .header--index {
-        grid-template-columns: 1fr;
+        grid-template-rows: auto auto;
+        grid-template-columns: 0.75fr 0.25fr;
```

```suggestion
        grid-template-columns: 3fr 1fr;
```

Я тут подумал, а это не одно и то же будет? По сути, это доли: 3 к 1. Так кажется проще читается.

---

Да, точно